src/components/Sum.js

```diff
@@ -0,0 +1,23 @@
+import { number } from 'prop-types';
```

Comment:
Could you explain to me why you use `import { number } from 'prop-types';` instead `import PropTypes from 'prop-types'; ?